frontend/simple/views/CreateGroup.vue

```diff
@@ -3,188 +3,78 @@
-                        <option>EUR</option>
-                      </select>
-                    </span>
-                  </p>
-                  <p class="control">
-                    <input class="input" type="text" name="incomeProvided"
-                      placeholder="Amount"
-                      data-vv-as="Income Provided"
-                      v-validate data-vv-rules="required|decimal:2"
-                      v-model="incomeProvided"
-                    />
-                  </p>
-                </div>
-                <i18n v-if="errors.has('incomeProvided')" data-control="incomeProvided" class="help is-danger">
-                  The Income Provided field must be a numeric currency amount and may contain 2 decimal points.
-                </i18n>
-              </div>
-              <div class="box" >
-                <p class="title is-5"><i18n>How transparent should your group be about who contributes?</i18n></p>
-                <p class="select">
-                  <select
-                    v-validate
-                    data-vv-rules="required"
-                    data-vv-as="Contribution Privacy"
-                    name="contributionPrivacy"
-                    v-model="contributionPrivacy"
-                  >
-                    <option value="">Select an option</option>
-                    <option value="Very Private">Very Private</option>
-                  </select>
-                </p>
-                <i18n v-if="errors.has('contributionPrivacy')" data-control="contributionPrivacy" class="help is-danger">
-                  {{ errors.first('contributionPrivacy') }}
-                </i18n>
-              </div>
-              <div class="has-text-centered button-box">
-                <div id="errorMsg" v-if="errorMsg" class="help is-danger">{{ errorMsg }}</div>
-                <button
-                  class="button is-success is-large"
-                  type="submit"
-                >
-                  <i18n>Create Group</i18n>
-                </button>
-              </div>
-            </div>
+      <div class="column is-half is-offset-one-quarter" >
+        <transition name="fade" mode="out-in">
+          <router-view :group="form" :validity="validity" @input="(payload) => updateGroupData(payload)">
+          </router-view>
+        </transition>
+
+        <div class="field step-controls">
+          <p class="control" v-if="currentStep + 1 < config.steps.length">
+            <button
+              class="button is-success is-large"
+              @click.prevent="next"
+              id="nextBtn"
+            >
+              <i18n>Next</i18n>
+              <span class="icon">
+                <i class="fa fa-arrow-right"></i>
+              </span>
+            </button>
+          </p>
+          <p class="control" v-if="currentStep + 1 === config.steps.length">
+            <button
+              class="button is-success is-large"
+              @click.prevent="submit"
```

Comment:
This `.prevent` thing is something I asked Tamas about as well.

Is it necessary or will this work without it without causing any problems? My reading of the [event docs](https://vuejs.org/v2/guide/events.html) suggests that `.prevent` is mostly useful in dealing with forms (preventing submit buttons from reloading the page).

If it's not necessary here or on line 16, then remove it.

Reply:
It was necessary while this was all wrapped in a form, but you are right, I removed the `form` so they are not necessary anymore. removing 👍